src/examples/__init__.py

```
@@ -29,4 +29,5 @@
 from . import base
 
 from .app import MBApp
-from .base import get_api
+from .app_v2 import MBAppV2
```

Ruff (0.8.0)

`.app_v2.MBAppV2` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)